src/TLuaInterpreter.cpp

```diff
@@ -2548,6 +2548,21 @@ int TLuaInterpreter::disableScrolling(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#scrollingState
+int TLuaInterpreter::scrollingState(lua_State* L)
```

Comment:
Do we need a new function for this or would getConfig be enough? @Mudlet/lua-interface

Reply:
If you move it to `getConfig(...)` you'd then have to convert the **already published *setters*** to be done via `setConfig(...)` instead - thereby breaking our backwards compatibility promise **a bit**!

Reply:
Yep, fair.

Reply:
The other thought was to call it scrollingActive to be in alignment with isActive? if scrollingActive('window') then ...

Reply:
Well `isActive(...)` is about counting the number of matching Mudlet items of a particular type that are (supposedly) able to respond to their particular input whereas this function is about reporting whether a particular user/package generated `TConsole` will scroll when new content is appended. To me there is no similarity of function so aligning the name of this new function with the former is not helpful...

:man_shrugging:

Reply:
It is doing a similar thing because it's checking the state of something

Reply:
So it should be `isScrollingActive([windowName])` then? :stuck_out_tongue_winking_eye: Fortunately it doesn't look like we provide *getters* for the visibility of the vertical or horizontal scroll-bars so there is no danger of confusing things there...

Reply:
Just `if scrollingActive('window') then`, I think that is most readable

Reply:
Okay, will do...